documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -1,54 +1,125 @@
-- [ ] run `make gen && make docsgen-cli` to generate documentation
-- [ ] create a **PR** targetting `release/vX.Y.Z` or `release/miner/vX.Y.Z` branch
-  - Opening a PR will trigger a CI run that will build the release and publish it to GitHub as a draft
-  - Merging the PR will trigger a CI run that will publish the GitHub release (it will also create a git tag)
+[//]: # (Copy/paste this for each RC, and increment "X")
+### RCX
+- [ ] To Be Added
 
-**Testing**
+### Stable (non-RC)
+- [ ] To Be Added
 
-Test the release candidate thoroughly, including automated and manual tests to ensure stability and functionality across various environments and scenarios.
-
-**Stable Release**
-
-Perform the following changes to the `release/vX.Y.Z` branch through a PR:
-- [ ] update the version string in `build/version.go` to one **NOT** ending with '-rcX'
-- [ ] run `make gen && make docsgen-cli` to generate documentation
-- [ ] create a **PR** targetting `release/vX.Y.Z` or `release/miner/vX.Y.Z` branch
-  - Opening a PR will trigger a CI run that will build the release and publish it to GitHub as a draft
-  - Merging the PR will trigger a CI run that will publish the GitHub release (it will also create a git tag)
-
-**Post-Release**
+## ✅ Release Checklist
 
-- [ ] Open a pull request against the `master` branch with a merge of the `releases` branch. Conflict resolution should ignore the changes to `version.go` (keep the `-dev` version from master). Do NOT delete the `releases` branch when doing so!
-- [ ] Update [RELEASE_ISSUE_TEMPLATE.md](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md) with any improvements determined from this latest release iteration.
-- [ ] Create an issue using [RELEASE_ISSUE_TEMPLATE.md](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md) for the _next_ release.
+### Before RC1
+- [ ] (network upgrade) Make sure all [Lotus dependecies are updated to the correct versions for the network upgrade](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/Update_Dependencies_Lotus.md)
+   - Link to PR:
+- [ ] Open PR against [RELEASE_ISSUE_TEMPLATE.md](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md) with title `docs(vX.Y.Z): release template improvements` for improving future releases.
+   - Link to PR:  
+   - This will get merged in a `Post Release` step, but improvements are better done by collecting notes along the way rather than just thinking about it at the end.
+- [ ] Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from `master` and make any further release-related changes to this branch.
+- [ ] In the `master` branch, bump the version in `build/version.go`  to `vX.Y.(Z+1)-dev`. Run `make gen && make docsgen-cli` before committing changes.
```

Comment:
I think we can make it clearer here that the version string in:
- https://github.com/filecoin-project/lotus/blob/4b75fa86b2567409b815ac4627256bd275e65fa3/build/version.go#L10 should be updated if shipping a node release.
- https://github.com/filecoin-project/lotus/blob/4b75fa86b2567409b815ac4627256bd275e65fa3/build/version.go#L21 should be updated if shipping a miner release.

Reply:
Actually, since this is the `master` we should be fine with saying something like "bump the version for the type of release you are shipping node|miner"

Reply:
Good point - addressed in https://github.com/filecoin-project/lotus/pull/12356/commits/3484d63c1eb1a781d917b8f442bb29136a031fbc